regression-test/suites/nereids_syntax_p0/test_function_regexp_passall.groovy

```diff
@@ -0,0 +1,53 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+suite("test_function_regexp_passall") {
+    def tableName = "test_function_regexp_passall"
+    sql """
+        CREATE TABLE ${tableName} (
```

Comment:
do not use `${tabelName}`, just use `test_function_regexp_passall` directly, this will make reproduce test case more easily.

Reply:
@zhiqiang-hhhh  Done. Thanks for your suggestion ;)